creator-node/test/lib/helpers.js

```diff
@@ -31,16 +31,16 @@ const uploadTrack = async (filePath, cnodeUserUUID, blacklistManager) => {
   return resp
 }
 
-const saveFileToStorage = (filePath) => {
-  const file = fs.readFileSync(filePath)
+const saveFileToStorage = async (filePath) => {
+  const file = await fs.readFile(filePath)
   const fileName = uuid()
   const fileDir = path.join(
-    DiskManager.getTmpTrackUploadArtifactsPath(),
+    await DiskManager.getTmpTrackUploadArtifactsPath(),
     fileName
   )
-  fs.mkdirSync(fileDir)
-  fs.mkdirSync(fileDir + '/segments')
-  fs.writeFileSync(path.join(fileDir, `${fileName}.mp3`), file)
+  await fs.mkdir(fileDir)
+  await fs.mkdir(fileDir + '/segments')
```

Comment:
could use mkdirp / mkdirs - but this is fine too

Reply:
ya dont care since tests